actions/setup-crib-environment/action.yml

```diff
@@ -152,7 +159,19 @@ runs:
 
         nix develop -c ./cribbit.sh "$NAMESPACE"
 
-        nix develop -c devspace deploy --skip-build -o=${{ inputs.image-tag }}
+        profile_args_array=()
+        if [[ -n "$PROFILES" ]]; then
+          # Convert comma-separated string to array
+          IFS=',' read -r -a profiles_array <<< "$PROFILES"
+
+          for profile in "${profiles_array[@]}"; do
+            profile_args_array+=("-p")
+            profile_args_array+=("$profile")
+          done
+          echo "running devspace with profile args: ${profile_args_array[@]}"
+        fi
+
+        nix develop -c devspace deploy --skip-build -o=${{ inputs.image-tag }} "${profile_args_array[@]}"
```

Comment:
Initially I had it in the separate script, but it didn't work when passing as string. It only works if we pass this as an array